src/ChunkSplitters.jl

```diff
@@ -9,30 +9,45 @@ export chunks, getchunk
 
 This function returns an iterable object that will split the *indices* of `array` into
```

Comment:
Either "indices" or "indexes" is fine but we should be consistent about it. Here, we say "indices" (which I personally like better) but in the new examples you use `idxs`. Should either be `idcs` there or "indexes" here (and potentially elsewhere).

Reply:
Do you prefer `idcs` or `inds`?

Reply:
In Julia Base the convention is to use `indices` and `inds`, so I'll keep that.